Source/spelldat.h

```diff
@@ -21,63 +21,63 @@ enum spell_type : uint8_t {
 	RSPLTYPE_INVALID,
 };
 
-enum spell_id : int8_t {
-	SPL_NULL,
-	SPL_FIREBOLT,
-	SPL_HEAL,
-	SPL_LIGHTNING,
-	SPL_FLASH,
-	SPL_IDENTIFY,
-	SPL_FIREWALL,
-	SPL_TOWN,
-	SPL_STONE,
-	SPL_INFRA,
-	SPL_RNDTELEPORT,
-	SPL_MANASHIELD,
-	SPL_FIREBALL,
-	SPL_GUARDIAN,
-	SPL_CHAIN,
-	SPL_WAVE,
-	SPL_DOOMSERP,
-	SPL_BLODRIT,
-	SPL_NOVA,
-	SPL_INVISIBIL,
-	SPL_FLAME,
-	SPL_GOLEM,
-	SPL_BLODBOIL,
-	SPL_TELEPORT,
-	SPL_APOCA,
-	SPL_ETHEREALIZE,
-	SPL_REPAIR,
-	SPL_RECHARGE,
-	SPL_DISARM,
-	SPL_ELEMENT,
-	SPL_CBOLT,
-	SPL_HBOLT,
-	SPL_RESURRECT,
-	SPL_TELEKINESIS,
-	SPL_HEALOTHER,
-	SPL_FLARE,
-	SPL_BONESPIRIT,
-	SPL_LASTDIABLO = SPL_BONESPIRIT,
-	SPL_MANA,
-	SPL_MAGI,
-	SPL_JESTER,
-	SPL_LIGHTWALL,
-	SPL_IMMOLAT,
-	SPL_WARP,
-	SPL_REFLECT,
-	SPL_BERSERK,
-	SPL_FIRERING,
-	SPL_SEARCH,
-	SPL_RUNEFIRE,
-	SPL_RUNELIGHT,
-	SPL_RUNENOVA,
-	SPL_RUNEIMMOLAT,
-	SPL_RUNESTONE,
+enum class SpellID : int8_t {
+	Null,
+	Firebolt,
+	Healing,
+	Lightning,
+	Flash,
+	Identify,
+	FireWall,
+	TownPortal,
+	StoneCurse,
+	Infravision,
+	Phasing,
+	ManaShield,
+	Fireball,
+	Guardian,
+	ChainLightning,
+	FlameWave,
+	DoomSerpents,
+	BloodRitual,
+	Nova,
+	Invisibility,
+	Inferno,
+	Golem,
+	Rage,
+	Teleport,
+	Apocalypse,
+	Etherealize,
+	ItemRepair,
+	StaffRecharge,
+	TrapDisarm,
+	Elemental,
+	ChargedBolt,
+	HolyBolt,
+	Resurrect,
+	Telekinesis,
+	HealOther,
+	BloodStar,
+	BoneSpirit,
+	LastDiablo = BoneSpirit,
+	Mana,
+	Magi,
+	Jester,
+	LightningWall,
+	Immolation,
+	Warp,
+	Reflect,
+	Berserk,
+	RingFire,
+	Search,
+	RuneFire,
+	RuneLightning,
+	GreaterRuneLightning,
+	GreaterRuneFire,
+	RuneStone,

```

Comment:
should be "RuneOf" instead of "Rune"

Reply:
Applied same principle to all things where "Of" was omitted